docs/operation/disaster_recovery/bootstrap_backup_restore.rst

```diff
@@ -0,0 +1,96 @@
+Bootstrap Node Backup and Restoration
+=====================================
+
+This section describes how to manually back up a MetalK8s bootstrap node,
+and how to restore a bootstrap node from such backup.
+
+.. note::
+
+   A backup is run automatically:
+
+   - at the end of the bootstrap,
+
+   - at the beginning of the upgrade/downgrade,
+
+   - at the end of the upgrade/downgrade,
+
+   - at the end of a bootstrap restoration.
+
+Backing up a Bootstrap Node
+***************************
+
+A backup file is generated at the end of the bootstrap.
+
+To create a new backup file run the following command:
+
+.. code::
+
+    /srv/scality/metalk8s-X.X.X/backup.sh
+
+Backup archives are stored in ``/var/lib/metalk8s/``.
+
+Restoring a Bootstrap Node
+**************************
+
+.. warning::
+
+   You must have a highly available control plane with at least
+   3 members in the ``etcd`` cluster (including the failed bootstrap node),
+   to use the restore script.
+
+.. note::
+
+   To restore a bootstrap node you need a backup archive and MetalK8s ISOs.
+   All the ISOs referenced in the bootstrap configuration file (located in
+   ``/etc/metalk8s/bootstrap.yaml``) must be present.
+
+#. Unregister the unreachable ``etcd`` member from the cluster by running
+   the following commands from a working node with the ``etcd`` role:
+
+   .. code::
+
+      # Get etcd container id
+      CONT_ID=$(crictl ps -q --label io.kubernetes.container.name=etcd --state Running)
+
+      # List all etcd members to get the ID of the etcd member that need to be removed
+      crictl exec -it "$CONT_ID" \
+         etcdctl --endpoints https://localhost:2379 \
+         --cacert /etc/kubernetes/pki/etcd/ca.crt \
+         --key /etc/kubernetes/pki/etcd/server.key \
+         --cert /etc/kubernetes/pki/etcd/server.crt \
+         member list
+
+      # Remove the etcd member (replace <etcd_id> in the command)
+      crictl exec -it "$CONT_ID" \
```

Comment:
We have $variables and <variables>. We should agree on which to use.
@gdemonet  IIRC I received feedback on the RING's documentation, saying {{Variables}} are not convenient when copy-pasting commands. $variable can be set and automated prior to copy pasting the commands right?

Also, would it make sense to have a page in the docs listing all the variables used in here?

Reply:
In this specific context, we're asking the user to replace a single `<variable>`. The `$CONT_ID` one is not for the user to change, since it is set in a previous command.

If we were to use the same syntax, e.g. `$etcd_member_id` instead, and this variable was not set, how could the user know which part of the command needs to be manually edited?

Also, note that when doing variable expansion in a shell, unset variables are replaced by an empty string, which could induce strange behaviour from the copy-pasted commands.

IMO, for such cases, we should either:
- get rid of the manual replacement step, if possible (e.g. by retrieving the value from a previous command, and setting a variable to hold it)
- or keep it like this, because sometimes it's not even possible (and in this case, I think we're fine with asking input from the user)

Long story short, I think we shouldn't change anything here. And yes, user input is not super convenient when copy-pasting, but how could this be improved? Is it better to split the command in two, like:

```shell
USER_INPUT="<place your value here>"
./do-something-with "$USER_INPUT"
```

? Honestly not sure, but if you feel strongly about it, it can be done.

> would it make sense to have a page in the docs listing all the variables used in here?

What for? Who would need this? Such placeholders are only useful in the context where they are used, their names may be re-used in different places for conciseness, and they cannot be "automatically set" (otherwise, we would provide commands to do so).